### ISSUE TITLE
Passwordless signin doesn't work after updating to 2024.2

### ISSUE THREAD

**Issue by b-** [goauthentik/authentik#8636]
**Describe the bug**
I set up a passwordless authentication flow by following [this guide](https://goauthentik.io/docs/flow/stages/authenticator_validate/#passwordless-authentication), but it fails with an exception since updating to 2024.2.

**To Reproduce**
1. Install Authentik <2024.2 (I think I had 2023.6 installed when creating this auth flow, and I've updated it regularly within a few days of release.)
2. Create a passwordless flow with designation set to authentication, as per [this guide](https://goauthentik.io/docs/flow/stages/authenticator_validate/#passwordless-authentication)
3. Update to Authentik 2024.2

**Expected behavior**
Passwordless authentication should still work

**Screenshots**
<img width="1263" alt="image" src="https://github.com/goauthentik/authentik/assets/284789/22e8c538-22d3-49ff-bf54-8cdf9bb84ef6">

**Logs**
<details>
    <summary>Stacktrace from authentik</summary>

```
Traceback (most recent call last):
  File "/authentik/flows/views/executor.py", line 291, in get
    stage_response = self.current_stage_view.dispatch(request)
                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/ak-root/venv/lib/python3.12/site-packages/django/views/generic/base.py", line 143, in dispatch
    return handler(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/authentik/stages/authenticator_validate/stage.py", line 243, in get
    challenges = self.get_webauthn_challenge_without_user()
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/authentik/stages/authenticator_validate/stage.py", line 216, in get_webauthn_challenge_without_user
    "challenge": get_webauthn_challenge_without_user(
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/authentik/stages/authenticator_validate/challenge.py", line 72, in get_webauthn_challenge_without_user
    return loads(options_to_json(authentication_options))
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/ak-root/venv/lib/python3.12/site-packages/webauthn/helpers/options_to_json.py", line 118, in options_to_json
    auth_to_return["userVerification"] = options.user_verification.value
                                         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
builtins.AttributeError: 'str' object has no attribute 'value'
```
</details>

**Version and Deployment (please complete the following information):**
- authentik version: 2024.2.0
- Deployment: docker-compose

**Additional context**
I exported my "Passwordless" flow and uploaded it to [this Gist](https://gist.github.com/b-/c72d796139c387c1d859cd5e6781ec3b)

**Comment by bentschrader** [comment]
Exactly the same behaviour at my installation.

- authentik version: 2024.2.0
- Deployment: docker-compose

**Comment by ServusNL** [comment]
Same here, using passkey as a second factor does work but the passwordless flow breaks with above error.

**Comment by yeyeoke** [comment]
Same issue here, worked fine before upgrading to 2024.2.0.

**Comment by BeryJu** [comment]
duplicate of #8618, will be fixed by 2024.2.1
